src/corretores/corretores_unificado.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import numpy as np
+from auxilio.variaveis import (codigo_2lingua_dicionario as c2l,
+                               nome_col_df_respostas_enem as ncenem,
+                               nome_col_df_respostas_simulinho as ncsimulinho)
+
+
+def correcao_simulinho(df_respostas, df_gabarito, tipo_correcao):
+
+    lista_dfs_2lingua = get_pares_dfs_2lingua(df_respostas, df_gabarito, tipo_correcao)
+    lista_dfs_modelados = modela_df_respostas(lista_dfs_2lingua, tipo_correcao)
+    lista_dfs_modelados = add_verificacao(lista_dfs_modelados)
+
+    df_resultados = pd.concat(lista_dfs_modelados)
+    df_resultados.sort_index(inplace=True)
+
+    return df_resultados
+
+
+def correcao_enem(df_respostas, df_gabarito, tipo_correcao):
+
+    lista_dfs_2lingua = get_pares_dfs_2lingua(df_respostas, df_gabarito, tipo_correcao)
+    lista_dfs_modelados = modela_df_respostas(lista_dfs_2lingua, tipo_correcao)
+    lista_dfs_modelados = add_verificacao(lista_dfs_modelados)
+
+    df_resultados = pd.concat(lista_dfs_modelados)
+    df_resultados.sort_index(inplace=True)
+
+    return df_resultados
+
+# TODO: Adicionar UFSC
+def correcao_ufsc():
+    pass
+
+
+# Auxilio TODO: Talvez mover para arquivo externo?
+def get_pares_dfs_2lingua(df_respostas, df_gabarito, tc):
```

Comment:
@matos1396 o que seria tc? Espero que não seja o que eu to pensando o que é 😳, poderia por favor adicionar em um glossário ou adicionar um comentário explicando?